### DISCUSSION TITLE
Is there way to not show the minimap for smaller files?

### DISCUSSION THREAD

**Discussion by mfn** [Nasller/CodeGlancePro#147] [Ideas]
I see that setting feature e.g. "Max lines count", but I'm looking for the reverse, something like "Min lines count": files having less lines than "Min lines count" should not show the minimap.

For me, for files having < 200 I don't need a minimap and it would be great it could be automatically hidden.

Thanks, great plugin 🙇🏼

**Comment by Nasller** [comment]
you can have one.wait some time

**Comment by Nasller** [comment]
please download the EAP version, from this https://plugins.jetbrains.com/plugin/18824-codeglance-pro/versions/eap

**Reply by mfn** [reply]
are you kidding 😅 that's insane and it works perfectly
![image](https://github.com/user-attachments/assets/b312e5bd-3525-4e33-826c-d98b5247ede7)

**Reply by Nasller** [reply]
it is easy to me, so i can add this feature so fast. :)
